docs/reference/commandline/node_update.md

```diff
@@ -23,6 +23,14 @@ Options:
       --role string           Role of the node (worker/manager)
 ```
 
+This command is a *low-level* command to update membership, role and
+availability of a node. Each of these attributes can be updated using
+a simpler command such as `accept` for membership, `promote` or
+`demote` for role and `activate`, `drain` or `pause` for
+availability.
+
+This command must be run on a manager node, but may update any node in
+the swarm.
 
 ## Related information
```

Comment:
oh, and we need to update all the "related" sections to include the new commands